test/integrations/destinations/google_adwords_enhanced_conversions/dataDelivery/business.ts

```diff
@@ -6,8 +6,6 @@ import {
 } from '../../../testUtils';
 import { ProxyV1TestData } from '../../../testTypes';
 
-const API_VERSION = 'v18';
```

Comment:
version should be still present - & driven through version configured in integ-lib, why removing this ?

Reply:
Why we need version anymore as the SDK will contain the versions?

Reply:
As this is delivery flow, SDK is handling the endpoint generation, so yes, not required